libfulltext/response.py

```diff
@@ -4,17 +4,36 @@
 import hashlib
 import requests
 
+
 def assert_sha1(expected_sha1, expected_path):
-    """Returns a function that checks a save_response call via expected SHA1 hash and path"""
+    """Returns a function that checks a save_response call via expected SHA1 hash and path
+
+    Args:
+        expected_sha1: expected SHA1 hash of the downloaded file
+        expected_path: expected basename of the downloaded file
+
+    Returns:
+        A function that takes a stream (from a response) and a string
+        (basename where the stream should usually be saved). To match the
+        interface of save_stream
```

Comment:
The last sentence is kind of a half-sentence ;)

Reply:
changed to "This function matches the interface of save_stream."